test/unit/base_cli_test.rb

```diff
@@ -13,7 +13,7 @@
       Inspec::BaseCLI.stubs(:default_options).returns(default_options)
 
       opts = cli.send(:merged_opts, :exec)
-      expected = { 'format' => 'json', 'backend_cache' => false }
+      expected = { 'format' => 'json', 'backend_cache' => false, 'reporter' => { 'cli' => nil } }
```

Comment:
I think we should not have json and cli reporter enabled at the same time

Reply:
Can you explain why?

Reply:
@chris-rock actually having them both active is one of the primary use cases, where we want the console output in Jenkins or gitlab CI/CD to record the CLI output and we want to produce an artifact json of the results that can be saved / processed. Or a user is running a kitchen job on their laptop and they want to see the CLI but also need to produce JSON to push to someone else or another system.